terraform/pipeline/deploy.yml

```diff
@@ -5,11 +5,11 @@ parameters:
     type: string
 
 steps:
-  # https://github.com/microsoft/azure-pipelines-terraform/tree/main/Tasks/TerraformInstaller#readme
-  - task: TerraformInstaller@0
+  # https://github.com/microsoft/azure-pipelines-terraform/blob/main/Tasks/TerraformInstaller/TerraformInstallerV1/README.md
+  - task: TerraformInstaller@1
```

Comment:
this one was a clean swap. the TerraformTask upgrade is going to require a little bit of spelunking.